ncm-dirperm/src/main/pan/components/dirperm/schema.pan

```diff
@@ -30,11 +30,13 @@ function dirperm_permissions_valid = {
 };
 
 type structure_dirperm_entry = {
-    'path'    : string with match(SELF, '^/')
-    'perm'    : string with match(SELF,'[0-7]{3,4}')
-    'owner'   : string with match(SELF, '\w+(:\w+)?')
-    'type'    : string with match(SELF, 'd|f')
-    'initdir' ? string[]
+    'path'         : string with match(SELF, '^/')
+    'perm'         : string with match(SELF,'[0-7]{3,4}')
+    'owner'        : string with match(SELF, '\w+(:\w+)?')
+    'type'         : string with match(SELF, '^[df]$')
+    'initdir'      ? string[]
+    'checkmount'   : boolean = false
```

Comment:
can you add some documentation as follows
```pan
    @{does X}
    'checkmount'   : boolean = false
    @{behaves like Y}
    'within_mount' : boolean = false
```

(the `@{}` has to come before the attribute that it refers to)